src/dvclive/dvc.py

```diff
@@ -131,9 +132,14 @@ def _update_entries(old, new, key):
 def get_exp_name(name, scm, baseline_rev) -> str:
```

Comment:
Updated `get_exp_name` to always be able to return an experiment name instead of only when inside a repo.